_osx_setup.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
```

Comment:
ok i think i see what you're trying to do here. are you trying to indicate that this file is 'private' in a sense, and shouldn't be run directly by users? personally i think it's completely valid to expect a user to run two different shell scripts. one for setup, one to start/run the machine, especially since the setup script would be need far less often than the start/run scripts